search/geocoder.cpp

```diff
@@ -219,15 +222,15 @@ MwmSet::MwmHandle FindWorld(Index const & index, vector<shared_ptr<MwmInfo>> con
   return handle;
 }
 
-strings::UniString AsciiToUniString(char const * s) { return strings::UniString(s, s + strlen(s)); }
+UniString AsciiToUniString(char const * s) { return UniString(s, s + strlen(s)); }
```

Comment:
No need in the second const - I'm not planning to reassign pointer. You write
```c++
void Foo(int a, double b);
```
not
```c++
void Foo(int const a, double const b);
```
right? Here is the same thing.